src/index.html

```diff
@@ -9,6 +9,86 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="header__link"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="header__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#phone"
+              class="nav__link"
+            >
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#pc"
+              class="nav__link"
+              data-qa="hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#gadget"
+              class="nav__link"
+            >
+              Гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#tabletpc"
+              class="nav__link"
+            >
+              Планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              Фото
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
   </body>
-</html>
+</html>
```

Comment:
Always add newline at the end of each file
```suggestion
</html>

```